aptos-move/move-examples/fungible_asset/sources/coin_example.move

```diff
@@ -0,0 +1,136 @@
+module aptos_framework::coin_example {
```

Comment:
I wonder if we should turn this example into a NoCodeCoin? Fungible source + managed_fungible_source are all extensions so we still need something to call the underlying object creation + their init functions to fully initialize a new coin type. This is what the example here is doing

Reply:
yes, I will rename it.